.eslintrc.json

```diff
@@ -53,7 +53,11 @@
               "**/*.stories.tsx"
             ]
           }
-        ]
+        ],
+        "no-shadow": "off",
+        "@typescript-eslint/no-shadow": ["error"],
+        "no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": "error"
```

Comment:
i was getting enum declared but not used error even after using it.
Found this configuration as solution to it.
https://github.com/typescript-eslint/typescript-eslint/issues/2621#issuecomment-701970389